### ISSUE TITLE
Did the "Save as HTML" option disappear?

### ISSUE THREAD

**Issue by mcolegro** [madskristensen/MarkdownEditor2022#40]
I was using the "Save as HTML" option.  I can't seem to find it.  Is it still available?

**Comment by madskristensen** [comment]
How should it work this time around? 

Right-click foo.md file and hit Save as HTML to produce foo.md.html and nest it under foo.md? Then update foo.md.html every time foo.md is edited?

**Comment by mcolegro** [comment]
I thought the original implementation was pretty clean and straight forward.  Nesting the .html file under the .md file is fine, but I don't think the .md was carried over to the nested html filename.  Foo.md -> foo.html

**Comment by madskristensen** [comment]
@mcolegro ok, makes sense. So just like it was in the old extension? No improvements needed?

**Comment by mcolegro** [comment]
I think the way it works in the earlier version is pretty simple and straight forward.

**Comment by isbeorn** [comment]
The generation in the old version was nice. would be great if v2 could have it back.

**Comment by paulo-guimaraes** [comment]
Without "Save as HMTL", at the moment, I need to use an external tool such as notepad++ to convert to HTML. Please add this function back.

**Comment by tmoskaluk** [comment]
Are there any plans to fix this issue soon?

**Comment by mcolegro** [comment]
I switched back to the 1.x version to continue using the save to Html function.  I'll upgrade once the functionality has been added to the 2.x version.

**Comment by danielabbatt** [comment]
Yeah I've had to downgrade as well as I need the .html generation, otherwise a great extension.
